hera_qm/utils.py

```diff
@@ -659,3 +659,38 @@ def dynamic_slice(arr, slice_obj, axis=-1):
     for ax in axis:
         slices[ax] = slice_obj
     return arr[tuple(slices)]
+
+
+def process_ex_ants(ex_ants=None, metrics_file=None):
+    """
+    Return list of excluded antennas from command line argument.
+    Input:
+       ex_ants -- comma-separated value list of excluded antennas
+       metrics_file -- file containing array info from hera_qm
+    Output:
+       list of excluded antennas
+    """
+    from . import metrics_io
```

Comment:
This is to circumvent complete circular dependence between utils and metrics_io right? @plaplant  is there a better way (other than this function having its own module)?

Reply:
Yeah, I don't think there's a way around burying the import here (which isn't to my personal coding taste) besides moving it to a different file. Could we just have `process_ex_ants` be in the `metrics_io` module? I know it's not a perfect fit, but it would remove the circular dependence issue.

Reply:
I like that solution best. I'll move it to metrics_io.

Reply:
:+1: